### ISSUE TITLE
Controls bug: Pause disables ability to move camera with keyboard

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#453]
I was pretty confused when I accidentally triggered this. I tried pressing every key chronologically to see if there was a way switch between base, raid and farms with the keyboard, and hit P for pause. Then I tried to move the camera with WASD and saw that nothing happened. I thought the game had glitched out - but a bit later I figured out that you need to drag with right click, and not left click to move the camera, so it wasn't completely stuck. Then I accidentally pressed H and then realized I had just paused the game.

TL;DR:

1. There should be an indication that the game is paused, e.g. a small dialog ("Game is paused \n Press 'P' to unpause.")
2. WASD shouldn't be disabled when in pause - unless perhaps the function of pause is to "lock" the keyboard? In this case, it should probably be called something that indicates this better.

**Comment by cpritchard** [comment]
+1 on this it can be confusing the first time you accidently pause the game and you think it has crashed/frozen

**Comment by hamburml** [comment]
Sorry for ignoring this issue.
Next patch there will be a game notification which also states how the game resumes.
